webapp/channels/src/components/integrations/installed_incoming_webhooks/installed_incoming_webhooks.tsx

```diff
@@ -128,6 +128,12 @@ export default class InstalledIncomingWebhooks extends React.PureComponent<Props
                         defaultMessage='No incoming webhooks match {searchTerm}'
                     />
                 }
+                hintText={
+                    <FormattedMessage
```

Comment:
since you are already wrapping hitnText with a span element, it is better if instead of use `<FormattedMessage />` (which generate another span wrapper, that will end up with `span > span`) component, you use the function `intl.formatMessage` (this one does not generate the span wrapper)